report-service/src/main/java/gov/cdc/dataingestion/nbs/converters/Hl7ToRhapsodysXmlConverter.java

```diff
@@ -266,7 +266,7 @@ private void validateHL7PATIENTRESULTType(List<HL7OrderObservationType> listOfOO
         for (HL7OrderObservationType ooType : listOfOOTypes) {
             HL7OBRType hl7OBRType = ooType.getObservationRequest();
             if ((null == hl7OBRType.getParent()) && (null == hl7OBRType.getParentResult())) {
-                if ((assumedParentOBRType != hl7OBRType) && (assumedChildOBRType != hl7OBRType)) {
+                if ((assumedParentOBRType != hl7OBRType) && (assumedChildOBRType != hl7OBRType) && assumedChildOBRType != null && assumedChildOBRType.getParent() != null) {
```

Comment:
[Non-Blocking + Nit picking] Just to be consistent, put all the conditions inside parentheses or nothing